qa/os/src/test/java/org/elasticsearch/packaging/test/SysVInitTests.java

```diff
@@ -0,0 +1,119 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.packaging.test;
+
+import org.elasticsearch.packaging.util.Platforms;
+import org.elasticsearch.packaging.util.ServerUtils;
+import org.elasticsearch.packaging.util.Shell;
+import org.junit.BeforeClass;
+
+import java.nio.file.Files;
+
+import static org.elasticsearch.packaging.util.FileUtils.assertPathsExist;
+import static org.hamcrest.CoreMatchers.anyOf;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assume.assumeTrue;
+
+public class SysVInitTests extends PackagingTestCase {
+
+    @BeforeClass
+    public static void filterDistros() {
+        assumeTrue("rpm or deb", distribution.isPackage());
+        assumeTrue(Platforms.isSysVInit());
+        assumeTrue(Platforms.isSystemd());
```

Comment:
Why do we need this check?

Reply:
It actually should be assumeFalse. This is necessary because some systems have both sysv init and systemd, where sysv commands are a thin layer over systemd. This assume is meant to ensure we only test on systems with true sysv init support. Without it the tests will fail as we are testing behavior specifically in our init.d script, which would not be run in the hybrid systems.